cumulusci/tasks/util.py

```diff
@@ -140,11 +145,22 @@ class FindReplace(BaseTask):
 
     def _init_options(self, kwargs):
         super(FindReplace, self)._init_options(kwargs)
+
         if "replace" not in self.options:
             self.options["replace"] = ""
         self.options["file_pattern"] = process_list_arg(
             self.options.get("file_pattern") or "*"
         )
+        self.options["env_replace"] = process_bool_arg(
+            self.options.get("env_replace") or False
+        )
+        if self.options["env_replace"]:
+            if self.options["replace"] in os.environ.keys():
+                self.options["replace"] = os.environ[self.options["replace"]]
```

Comment:
Actually getting the value from the environment should be done in _run_task, not _init_options. At least with MetaDeploy, it's possible for the task to run on a completely different system than where it was initialized, and we want to use the runtime environment variables.